pkg/image/bootc_disk.go

```diff
@@ -56,6 +56,10 @@ func (img *BootcDiskImage) InstantiateManifestFromContainers(m *manifest.Manifes
 		qcow2Pipeline.Compat = img.Platform.GetQCOW2Compat()
 		qcow2Pipeline.SetFilename(img.Filename)
 		return qcow2Pipeline.Export(), nil
+	case platform.FORMAT_VMDK:
+		vmdkPipeline := manifest.NewVMDK(buildPipeline, baseImage)
```

Comment:
This is nice! I would love to see a minimal test, maybe something like the attached diff?
[0001-image-add-minimal-test-for-vmdk.patch.txt](https://github.com/osbuild/images/files/14526554/0001-image-add-minimal-test-for-vmdk.patch.txt)

Ideally it would check a bit more but it might be a reasonable baseline

Reply:
Thanks, I've rolled that in!